WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -0,0 +1,84 @@
+<table class="table table-striped table-condensed admin-competitions">
+  <% tbody_class = title.include?("Recent") ? "" : "hide" %>
+  <thead>
+    <tr class="admin-table-header">
+      <th>
+        <%= "#{title} (#{competitions.count})" %>
+      </th>
+      <th class="admin-delegate">
+        Delegate(s)
+      </th>
+      <th class="admin-date">
+        Date
+      </th>
+      <th class="admin-date">
+        Announced
+      </th>
+      <th class="admin-date">
+        Report posted
+      </th>
+      <th class="admin-date">
+        Results posted
+      </th>
+      <th class="admin-button">
+        <span class="hide-div">(<%= tbody_class == "hide" ? "show" : "hide" %>)</span>
```

Comment:
Actually we can avoid this by doing 2nd lint like this:
```ruby
<% tbody_class = title.include?("Recent") ? "show" : "hide" %>
```
I know that we use this class below but adding class `show` isn't anything bad =)